tasks/main.yml

```diff
@@ -221,10 +221,16 @@
     - key: dyndns_force_tcp
       value: "{{ ad_dyndns_force_tcp | string }}"
     - key: dyndns_auth
-      value: "{{ 'GSS-TSIG' if ad_dyndns_auth else 'none' }}"
+      value: "{{ ad_dyndns_auth | string if ad_dyndns_auth else '' }}"
```

Comment:
So what does this write to the ini file?  something like `dyndns_auth = ` or `dyndns_auth = ''`?  Are these valid?  Is the intention that the role should write nothing to the ini file if `ad_dyndns_auth` is not set?

Reply:
With '' it should write nothing 'GSS-TSIG' will be applied as it is default in sssd.
I can now explicitly put there  'GSS-TSIG' or 'none' that would switch the authentication off